### ISSUE TITLE
Season Models & Current Data

### ISSUE THREAD

**Issue by jfacowns** [kyleskom/NBA-Machine-Learning-Sports-Betting#54]
First off, this is a really cool project and I’m really enjoying going through it. 

I do have a few questions however.

The repo includes data all the way back to the 2007-08 season but in the notes, you make reference to “the highest levels of validation accuracy when the training dataset started from the 2012 − 2013 season rather than from the 2007 − 2008 season.” 

Can I therefore assume that the model is using only 2012-13 until last year (2021-22) ? 

Also wondering what the reason might be for not including games that have already happened this season (2022-23)? Would this not be the most predictive data to use in a model given that players and coaches change teams every year? 

I’ve used the Get_data.py script to try and pull this years data after changing the year variables to 2022-23. The script runs well as expected and appears to be pulling data but once it’s finished, I see no output. Looks like it’s supposed to create a new season folder in Team-Data and then populate.

Again, this is really cool and I’m looking forward to seeing how this progresses. Shoutout to those who found a way to integrate different sportsbooks and pull odds. Awesome stuff.

**Comment by kyleskom** [comment]
1.) That note was added after I trained the models some time back, I added that from a paper I read to revise as some point in the future. The training data used goes back to 07.

2.) The way I originally implemented things its not super easy to automatically get must up to date date and retrain. Would like to do that at some point just wasn't a priority when first starting thing.

3.) get data should just save into the team data folder. It's not super clean and this is where a lot of the messiness / bad practices are.
